app/Http/Controllers/Api/Client/Servers/ScheduleTaskController.php

```diff
@@ -49,6 +50,10 @@ public function store(StoreTaskRequest $request, Server $server, Schedule $sched
             throw new ServiceLimitExceededException("Schedules may not have more than {$limit} tasks associated with them. Creating this task would put this schedule over the limit.");
         }
 
+        if ($server->backup_limit === 0 && $request->action === 'backup'){
+            throw new HttpForbiddenException("A backup task cannot be created when the backup limit for a server is set to {$server->backup_limit}.");
```

Comment:
This should return a bad request response, forbidden is used for bad tokens.  Message should be changed to something like `This server is not able to create any backups` or `A backup task cannot be created because the server has a backup limit of 0` (if you leave the message the same, you can replace `{$server->backup_limit}` with `0`.

Reply:
![image](https://user-images.githubusercontent.com/1757840/117086853-c2de4180-ad1b-11eb-9d2b-72ac3fb145dd.png)
![image](https://user-images.githubusercontent.com/1757840/117086857-c671c880-ad1b-11eb-8d25-82af6731232a.png)